docs/overview/private_key_management.md

```diff
@@ -0,0 +1,7 @@
+# Private Key Management
+
+LDK has "default implementations", but users can choose to provide private keys to LDK in any way they wish following a simple API. LDK even supports a generic API for signing transactions, allowing users to run LDK without any private keys in memory and/or putting private keys only on hardware wallets.
```

Comment:
Feel free to leave to a later clean-up, but would good to reword "users" as "you" wherever it makes sense.